pygmt/src/wiggle.py

```diff
@@ -30,7 +33,16 @@
     w="wrap",
 )
 @kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
-def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
+def wiggle(
+    self,
+    data=None,
+    x=None,
+    y=None,
+    z=None,
+    fill_positive=None,
```

Comment:
I think we should use `fillpositive` and `fillnegative` following the code styles (https://github.com/GenericMappingTools/pygmt/blob/main/doc/contributing.md#code-style):

> When wrapping a new alias, use an underscore to separate words bridged by vowels (aeiou), such as no_skip and z_only. Do not use an underscore to separate words bridged only by consonants, such as distcalc, and crossprofile. This convention is not applied by the code checking tools, but the PyGMT maintainers will comment on any pull requests as needed.

Reply:
Agree.